schematools/contrib/django/models.py

```diff
@@ -570,3 +572,14 @@ def related_model(self):
         dataset_name, table_name, column_name = [to_snake_case(part)
                                                  for part in self.relation.split(":")]
         return apps.all_models[dataset_name][table_name]
+
+
+def get_active_profiles():
+    profiles = cache.get(PROFILES_CACHE_KEY)
+    if profiles is None:
```

Comment:
Wordt de cache opnieuw gevuld bij een DSO-API restart? Dus dan met een change in de Profile zou bij een deploy de cache kunnen actualiseren toch?

Reply:
Ik heb gekozen om import aan te passen, om cache leeg te makken op het moment wanneer profielen zijn geladen.